plugin/callgraph/processors.go

```diff
@@ -272,7 +272,7 @@ func attributeProcessor(attributeNode *sitter.Node, treeData []byte, currentName
 
 	objectSymbol, attributeQualifierNamespace, err := attributeResolver(attributeNode, treeData, currentNamespace, callGraph, metadata)
 	if err != nil {
-		log.Errorf("Error resolving attribute - %v", err)
```

Comment:
Why are we not returning the error here or at least logging?

Reply:
hey, these errors are for cases that we're not yet handling (Not really errors, ig we can say ignored cases)
eg. resolving attribute for a function returned object - `someFunc().attr`
We get a LOT of these logs, Ideally these logs would be only useful when a developer is working directly on code repo
Hence commented